src/app/core/api.service.js

```diff
@@ -0,0 +1,39 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('xyz-cv-ui.core')
+        .factory('api', api);
+
+    function api($resource, config) {
```

Comment:
I'm not convinced about this design, but let's go with it for a while and we'll see how it turns out